guides/source/action_text_overview.md

```diff
@@ -8,149 +8,141 @@ rich text content.
+In order to install Action Text and start working with rich text content in your Rails app:
+
+1. Run `bin/rails action_text:install`
 
-NOTE: Action Text uses polymorphic relationships with the `action_text_rich_texts` table so that it can be shared with all models that have rich text attributes. If your models with Action Text content use UUID values for identifiers, all models that use Action Text attributes will need to use UUID values for their unique identifiers. The generated migration for Action Text will also need to be updated to specify `type: :uuid` for the `:record` `references` line.
+2. After the install command is complete, run `bin/rails db:migrate` to add the action_text and active_storage tables to your app
 
-After the installation is complete, a Rails app should have the following changes:
+The `action_text:install` command will do the following:
 
-1. Both `trix` and `@rails/actiontext` should be required in your JavaScript entrypoint.
+- Installs the Yarn packages for `trix` and `@rails/actiontext` and adds them to the `application.js`.
+- Adds an `image_processing` gem from Active Storage for image analysis and transformations of the embedded images and other attachments. Please refer to the Active Storage Overview guide for more information about the `image_processing` gem.
+- Adds migrations to create the following tables that store rich text content and attachments; namely `action_text_rich_texts`, `active_storage_blobs`, `active_storage_attachments`, `active_storage_variant_records`. Later on, we’ll discuss the polymorphic association of this table to your existing models.
+- Creates `actiontext.css` and imports it into `application.css`. The Trix stylesheet is also included in the `application.css` file.
+- Adds the default view partials `_content.html` and `_blob.html` to render Action Text content and Active Storage attachment (aka blob) respectively.
 
-    ```js
-    // application.js
-    import "trix"
-    import "@rails/actiontext"
-    ```
+When the Action Text installation creates the `action_text_rich_texts` table, it uses polymorphic relationships so that it can be shared with all your existing models through rich text attributes. This is done through a `record` column which stores the ClassName of your model and `record_id` which stores the relevant ID of your record.
 
-2. The `trix` stylesheet will be included together with Action Text styles in your `application.css` file.
+Hence, if your models containing Action Text content use UUID values as identifiers, then all models that use Action Text attributes will need to use UUID values for their unique identifiers. The generated migration for Action Text will also need to be updated to specify `type: :uuid` for the record references line.
 
 ## Creating Rich Text Content
 
-Add a rich text field to an existing model:
+To create and render rich text we need to follow some configurations. In this section, we’ll explore these configurations.
```

Comment:
These sentences could be simplified to: "This sections explores some of the configurations you'll need to follow in order to create and render rich text."